RetakesPlugin/Modules/Managers/GameManager.cs

```diff
@@ -11,17 +12,18 @@ public class GameManager
     public readonly QueueManager QueueManager;
     private readonly int _consecutiveRoundWinsToScramble;
     private readonly bool _isScrambleEnabled;
-
+    private readonly bool _RemoveSpectatorsEnabled;
     public const int ScoreForKill = 50;
     public const int ScoreForAssist = 25;
     public const int ScoreForDefuse = 50;
 
-    public GameManager(Translator translator, QueueManager queueManager, int? roundsToScramble, bool? isScrambleEnabled)
+    public GameManager(Translator translator, QueueManager queueManager, int? roundsToScramble, bool? isScrambleEnabled, bool? RemoveSpectatorsEnabled)
     {
         _translator = translator;
         QueueManager = queueManager;
         _consecutiveRoundWinsToScramble = roundsToScramble ?? 5;
         _isScrambleEnabled = isScrambleEnabled ?? true;
+        _RemoveSpectatorsEnabled = RemoveSpectatorsEnabled ?? true;
```

Comment:
You need to default this behaviour to false, as that's how it was before and I want to maintain backwards compatibility / not introduce a potentially breaking change to peoples servers.